### PR TITLE
Rubi integrator docs

### PR DESCRIPTION
**Pull request by vishalg2235**
Some Edits in about.rst file.

Fixes #13261

### CODE REVIEW

**Comment by vishalg2235** [comment]
@gxyd @Abdullahjavednesar please verify.

**Review by gxyd — COMMENTED** [review]
Try to limit each line to about ~79 characters. This is not a hard and fast rule, but would make review locally easier.

**Comment by vishalg2235** [comment]
@gxyd I have limited each line to 79 characters. 
A query: why everytime it says "Some checks haven't completed yet" ? How to pass that check ?

File: sympy/integrals/rubi/utility_function.py [hunk 1]
```diff
@@ -258,7 +261,7 @@ def IntegersQ(*var):
     return all(IntegerQ(i) for i in var)
 
 def ComplexNumberQ(*var):
-    """
+    r"""
```
Comment (gxyd) [inline sympy/integrals/rubi/utility_function.py]: You haven't addressed this comment https://github.com/sympy/sympy/pull/13261#discussion_r155200701

**Review by gxyd — COMMENTED** [review]
Please refer the original pull request in the pull request description. Also in the upcoming commit with 'fixes #original_pull_request_number'

**Comment by gxyd** [comment]
> A query: why everytime it says "Some checks haven't completed yet" ? How to pass that check ?

That would potentially mean that the travis ci tests are still running.

Also there was a failing unrelated test on travis, I have restarted it.

**Comment by vishalg2235** [comment]
@gxyd @Abdullahjavednesar please verify.

File: doc/src/modules/integrals/rubi/utility_functions.rst [hunk 2]
```diff
@@ -0,0 +1,142 @@
+Utility Functions
+=================
+
+Utility functions or helper functions plays an essential role in Rubi apart
```
Comment (gxyd) [inline doc/src/modules/integrals/rubi/utility_functions.rst]: 'plays' -> 'play'

File: doc/src/modules/integrals/rubi/about.rst [hunk 3]
```diff
@@ -0,0 +1,19 @@
+About Rule Based Integrator
+===========================
+
+About it
+--------
+
+A RUle Based Integrator nicknamed Rubi is module in sympy.integrals. It is an
+implementation of more than 6000 rules to cover a wide variety of indefinite
```
Comment (gxyd) [inline doc/src/modules/integrals/rubi/about.rst]: instead write:
It is an implementation of more than 6000 rules to determine antiderivative of a wide variety of mathematical expressions.

File: doc/src/modules/integrals/rubi/utility_functions.rst [hunk 4]
```diff
@@ -0,0 +1,142 @@
+Utility Functions
+=================
+
+Utility functions or helper functions plays an essential role in Rubi apart 
+from pattern matching. It is required to facilitate integration methods to
```
Comment (gxyd) [inline doc/src/modules/integrals/rubi/utility_functions.rst]: from -> for

**Comment by gxyd** [comment]
CC: @certik

File: doc/src/modules/integrals/rubi/about.rst (hunk 3 above)
Comment (Abdullahjavednesar) [reply doc/src/modules/integrals/rubi/about.rst]: May be we can write
A RUle Based Integrator nicknamed RUBI is..

Comment (vishalg2235) [reply doc/src/modules/integrals/rubi/about.rst]: I think its better to use 'RUBI' instead of 'Rubi' .

**Comment by vishalg2235** [comment]
@gxyd I have done corrections. 
PS. I have changed 'Rubi' to 'RUBI' everywhere. please give reviews?

**Comment by vishalg2235** [comment]
ping @gxyd.

**Comment by gxyd** [comment]
Let's wait for @certik and @Upabjojr , I think they were @Abdullahjavednesar 's mentor for RUBI module, I don't know the mathematical details here.

I would have expected the docs to be more descriptive, since I can't understand some of the things.

**Comment by vishalg2235** [comment]
yup!

**Comment by Upabjojr** [comment]
It's good to have some documentation, but I would recommend to unlink it from the doc generation root. We don't want this to be public, the rubi module is not finished.

File: doc/src/modules/integrals/rubi/index.rst [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+Contents
+========
+
+.. toctree::
+   :maxdepth: 2
+
+   about.rst
+   pattern_matcher.rst
+   utility_functions.rst
```
Comment (Upabjojr) [inline doc/src/modules/integrals/rubi/index.rst]: Missing a new line at the end of the file

**Comment by vishalg2235** [comment]
ok @Upabjojr . How to unlink it from the doc generation root ? I'm new to github I don't know how to do it.

**Comment by Upabjojr** [comment]
Make sure the docs don't get generated with this code.

**Comment by Upabjojr** [comment]
in the docs subdir there's a makefile. It should be make html to build locally

**Comment by vishalg2235** [comment]
@Upabjojr I didn't get how and what to do? Should I edit 'makefile' ffile? can you please explain.

**Comment by vishalg2235** [comment]
@Upabjojr if we make the files in some other directory instead of 'doc' then rubi docs will not be generated.
was you telling me something else to do for it?
**one question**: If we have the 'rubi' dir on same path in docs subdir then if we don't add its link in any rst file. will it still be linked from any webpage (mainly could be linked in 'Next topic') ? example: g-functions.rst is never linked in any rst file. but is linked at 'Next topic' in integrals.html

**Comment by vishalg2235** [comment]
Issue: I have generated a local copy of sympy doc using `make html` from master branch. But after I change my branch how to regenerate it to see changes in doc html ?

**Comment by vishalg2235** [comment]
ping @Upabjojr @gxyd @Abdullahjavednesar .

**Comment by czgdp1807** [comment]
@Upabjojr What's your view on this PR?

**Comment by Upabjojr** [comment]
We should have docs... unfortunately Rubi is still experimental, so maybe it's better not to make the docs definitive.

**Comment by czgdp1807** [comment]
Can we close this then?

**Comment by Upabjojr** [comment]
It's actually a pity to lose all of this effort in writing documentation for rubi... but it probably needs some updates.

**Comment by czgdp1807** [comment]
@Upabjojr Can we extend this and complete it? If it is so, then I will do it. Please let me know.

**Comment by certik** [comment]
What needs to be done to finish this PR?

**Comment by czgdp1807** [comment]
ping @Upabjojr Can I continue this? Please let me know what needs to be done to complete this PR. Thanks.

**Comment by Upabjojr** [comment]
Yes, you can. My only concern is that the `rubi` module is experimental. Users should be warned that the API may change unexpectedly between SymPy versions.

Furthermore, I would only mention this module as a nested note and make it explicit that this is only for technical purposes.

**Comment by czgdp1807** [comment]
Closing this PR. New one opened.

**Pull request closed without merging**